src/controlnet_aux/zoe/zoedepth/models/base_models/midas_repo/midas/backbones/beit.py

```diff
@@ -94,14 +94,43 @@ def attention_forward(self, x, resolution, shared_rel_pos_bias: Optional[torch.T
 def block_forward(self, x, resolution, shared_rel_pos_bias: Optional[torch.Tensor] = None):
     """
     Modification of timm.models.beit.py: Block.forward to support arbitrary window sizes.
+    Handles differences between timm 0.6.7 and 1.0+ attribute naming.
     """
-    if self.gamma_1 is None:
-        x = x + self.drop_path(self.attn(self.norm1(x), resolution, shared_rel_pos_bias=shared_rel_pos_bias))
-        x = x + self.drop_path(self.mlp(self.norm2(x)))
+    # Check which attributes exist to determine which version of timm we're using
+    if hasattr(self, 'gamma_1'):
+        # Original timm 0.6.7 structure
+        if hasattr(self, 'drop_path'):
+            # Original attribute name
+            drop_path_fn = self.drop_path
+        elif hasattr(self, 'drop_path1'):
+            # timm 1.0+ attribute name
+            drop_path_fn = self.drop_path1
+        else:
+            # Fallback to identity if neither exists
+            drop_path_fn = lambda x: x
```

Comment:
Seems like the if/else paths are the same except for the use of `self.gamma_1`? Also, does having `gamma_1` guarantee the presence of `gamma_2`?